.cirrus.yml

```diff
@@ -53,3 +53,27 @@ task:
   build_script:
     - export PATH=`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin:$PATH
     - ./script/incremental_build.sh build-examples --ipa
+
+task:
+  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
+  name: local_tests
+  osx_instance:
+    image: high-sierra-xcode-9.4.1
+  env:
+    PATH: $PATH:/usr/local/bin
+  setup_script:
+    - pod repo update
+    - git clone https://github.com/flutter/flutter.git
+    - git fetch origin master
```

Comment:
Doesn't make sense to me.  I'll remove it.

Reply:
Ok, it's necessary because of this line in the common.sh: https://github.com/flutter/packages/blob/bf86094ab154557b1e2ffafa5428791a1cd596ec/script/common.sh#L11